src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# Helper executable: generates parts of the sleef header file
+add_executable(${TARGET_MKRENAME} mkrename.c)
+target_compile_options(${TARGET_MKRENAME}
+  PRIVATE ${FLAGS_BASIC_C} ${FLAGS_WALL})
+
+# Helper executable: dispatcher
+add_executable(${TARGET_MKDISP} mkdisp.c)
+target_compile_options(${TARGET_MKDISP}
+  PRIVATE ${FLAGS_BASIC_C} ${FLAGS_WALL})
+
+# TODO: set the parameters for mkrename run
+set(MKRENAME_PARAMS "")
+
+configure_file(
+  ${CMAKE_CURRENT_SOURCE_DIR}/GenerateSleefHeader.cmake.in
+  ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake @ONLY)
+
+# Add sleeh header target
+add_custom_target(generate_sleef_header ALL
+  DEPENDS ${CMAKE_BINARY_DIR}/include/sleef.h)
+
+# Add custom command to process the sleef header and install it under /include
+add_custom_command(
+  OUTPUT ${CMAKE_BINARY_DIR}/include/sleef.h
+  COMMAND ${CMAKE_COMMAND} -P
+    ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake
+  DEPENDS
+    ${CMAKE_CURRENT_SOURCE_DIR}/sleeflibm.h.org
+    ${CMAKE_CURRENT_BINARY_DIR}/GenerateSleefHeader.cmake)
+
+# Install header files in the include directory
+add_custom_target(install_headers ALL
```

Comment:
This is indeed not very clear and could be quite confusing for someone picking up SLEEF development. 

Currently, all header files are put into the ${CMAKE_BUILD_DIR}/include and that is included via -I when compiling. I am doing this to avoid having to specify relative paths in the cmake files.  However, at installation time, only sleef.h is installed under ${CMAKE_INSTALL_DIR}/include.

I think we need to have a conversation about the header files sooner or later. I have a suggestion I am still working on and will submit a pull request after the SIMD version is building. But for now I think it's okay not to worry about the tree structure.